native/acter/src/api/verification.rs

```diff
@@ -699,13 +691,15 @@ async fn sas_verification_handler(
     event_id: Option<OwnedEventId>,
     txn_id: Option<OwnedTransactionId>,
     sender: OwnedUserId,
-) {
+) -> Result<()> {
     info!(
         "Starting verification with {} {}",
         &sas.other_device().user_id(),
         &sas.other_device().device_id()
     );
-    sas.accept().await.unwrap();
+    if let Err(e) = sas.accept().await {
+        return Err(e.into());
+    }
```

Comment:
Why not just ..

```suggestion
    sas.accept().await?;
```

Reply:
Fixed by https://github.com/acterglobal/a3/pull/1252/commits/f159e9e35eec03c6d91ecaff88d5b45562b67edc